src/tests/ftest/util/slurm_utils.py

```diff
@@ -244,9 +249,9 @@ def srun(nodes, cmd, srun_params=None):
         for key, value in srun_params.items():
             params_list.extend(["--{}={}".format(key, value)])
             params = " ".join(params_list)
-    cmd = "srun --nodelist={} {} {}".format(nodes, params, cmd)
+    cmd = ["srun", "--nodelist={}".format(nodes), params] + list(cmd)
```

Comment:
Do we need a full path for the `srun` command?

Reply:
not needed

Reply:
not needed